ec2/sc-ec2-linux-jumpcloud-v1.0.0.yaml

```diff
@@ -0,0 +1,488 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
Note file name. We should have a convention of adding a version number on the ends of the "LoadTemplateFromURL" templates.